### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: doc/POST_INSTALL.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ Your video files are stored by default in `/home/yunohost.app/tube/videos`.
```
Comment (alexAubin) [inline doc/POST_INSTALL.md]: I feel like it's probably more relevant to have this as an `ADMIN.md` ? (And also we probably can get rid of the "Configuration" title

**Comment by alexAubin** [comment]
>`"error opening store tube.db: mkdir tube.db: permission denied"`

Hmmm that's because there's no `w` permission on the install dir by default ... maybe we want to have `rwx` enabled for the owner by default ? (I mean in the core)

**Comment by ericgaspar** [comment]
I am testing with `__DATA_DIR__/tube.db` path

**Comment by ericgaspar** [comment]
or I can add `chmod 750 "$install_dir"`

edit: now we have `550` as default, right?

**Pull request merged**